pkg/performanceprofile/controller/performanceprofile_controller.go

```diff
@@ -422,6 +426,29 @@ func (r *PerformanceProfileReconciler) Reconcile(ctx context.Context, req ctrl.R
 		return reconcile.Result{}, nil
 	}
 
+	// Update the config.openshift.io/node object with the desired cgroupsv1 mode.
+	// (TODO) This code can be removed in the future when the cgroupsv2 is supported
+	key := types.NamespacedName{
+		Name: nodeCfgName,
+	}
+	nodeCfg := &apiconfigv1.Node{}
+	nodeCfg.SetGroupVersionKind(schema.GroupVersionKind{
+		Group:   "config.openshift.io",
+		Version: "v1",
+		Kind:    "Node",
+	})
+	err = r.Client.Get(context.Background(), key, nodeCfg)
+	if err != nil {
+		klog.Errorf("failed to get config node object; name=%q err=%v", nodeCfg.GetName(), err)
+		nodeCfg.Name = nodeCfgName
+		nodeCfg.Spec.CgroupMode = apiconfigv1.CgroupModeV1
+		r.Client.Update(ctx, nodeCfg)
```

Comment:
if we failed getting the object, don't we want to use Client.Create here ?

Reply:
Ideally yes. 
But in this case, the node config object is by default created, present in the cluster. Reference -> [here](https://github.com/openshift/cluster-config-operator/blob/master/empty-resources/0000_10_config-operator_02_node.cr.yaml) 
This error log is added just to be more cautious over any failure in getting the object.